### PR TITLE
Issue #85 update ISO template top-level to match SWOT

### PR DESCRIPTION
**Pull request by david-inglish**
These changes address Issue 85 for the top-level XML entries only: https://github.com/nasa/opera-sds-pge/issues/85

The time range change mentioned in the email will be made as part of another TBD issue.

### CODE REVIEW

File: src/opera/pge/templates/OPERA_ISO_metadata_L3_DSWx_HLS_template.xml.jinja2 [hunk 1]
```diff
@@ -1,864 +1,855 @@
...
+                                <eos:EOS_Processing>
+                                    <gmi:identifier>
                                         <gmd:MD_Identifier>
                                             <gmd:code>
-                                                <gco:CharacterString>{{ run_config.Groups.PGE.ProductPathGroup.ProductCounter }}{# ISO_OPERA_ProductCounter #}</gco:CharacterString>
+                                                <gco:CharacterString>PGEName: {{ catalog_metadata.PGE_Name }}{# ISO_OPERA_pgeName #} PGEVersion: {{ catalog_metadata.PGE_Version }}{# ISO_OPERA_PGEVersionId #}</gco:CharacterString>
                                             </gmd:code>
                                             <gmd:codeSpace>
-                                                <gco:CharacterString>gov.nasa.esdis.umm.otherid</gco:CharacterString>
+                                                <gco:CharacterString>gov.nasa.esdis.umm.pgeversionclass</gco:CharacterString>
```
Comment (collinss-jpl) [inline src/opera/pge/templates/OPERA_ISO_metadata_L3_DSWx_HLS_template.xml.jinja2:774]: 👍

Comment (collinss-jpl) [inline src/opera/pge/templates/OPERA_ISO_metadata_L3_DSWx_HLS_template.xml.jinja2:774]: 👍

**Review by collinss-jpl — COMMENTED** [review]
@david-inglish Changes to the template look great! Once the workarounds to `dswx_pge.py` are pruned from this branch, it should be good to merge.

File: src/opera/pge/templates/OPERA_ISO_metadata_L3_DSWx_HLS_template.xml.jinja2 [hunk 2]
```diff
@@ -1,864 +1,855 @@
...
+                            <gmd:description>
+                                <gco:CharacterString>OtherId: ProductCounter</gco:CharacterString>
+                            </gmd:description>
+                        </gmd:MD_Identifier>
+                    </gmd:identifier>
+                    <gmd:citedResponsibleParty>
                         <gmd:CI_ResponsibleParty>
                             <gmd:organisationName>
-                                <gco:CharacterString>Jet Propulsion Laboratory</gco:CharacterString>
+                                <gco:CharacterString>Jet Propulsion Repository</gco:CharacterString>
```
[the remark this answers is not in the document]
Comment (david-inglish) [reply src/opera/pge/templates/OPERA_ISO_metadata_L3_DSWx_HLS_template.xml.jinja2:219]: I think the display of the file differences is misleading here; the tool is trying to make associations where there are none.

I only changed the first "gmd:DS_Series" element to be "gmi:MI_Metadata", and then removed these 3 levels of XML elements to match the proposed output:

-    <gmd:composedOf>
-        <gmd:DS_DataSet>
-            <gmd:has>
-                <gmi:MI_Metadata>


-                </gmi:MI_Metadata>
-            </gmd:has>
-        </gmd:DS_DataSet>
-    </gmd:composedOf>

All of the lines between these removed fields was shifted left to fix indentation.

[the remark this answers is not in the document]
Comment (david-inglish) [reply src/opera/pge/templates/OPERA_ISO_metadata_L3_DSWx_HLS_template.xml.jinja2:219]: Line 34 in red at this location (old content) corresponds to line 30 in green (new content) if you scroll back up.  Only the indent has changed.

[the remark this answers is not in the document]
Comment (collinss-jpl) [reply src/opera/pge/templates/OPERA_ISO_metadata_L3_DSWx_HLS_template.xml.jinja2:219]: Ah I see now, this was always here. Looks like it got copied over from one of the NISAR templates I cribbed from.

**Review by collinss-jpl — APPROVED** [review]

**Pull request merged**